languages/elixir/exercises/concept/numbers/mix.exs

```diff
@@ -0,0 +1,28 @@
+defmodule FreelancerRate.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :numbers,
+      version: "0.1.0",
+      elixir: "~> 1.10",
```

Comment:
A previous decision was made to comment this out for backwards compatibility since some package manager versions are out of date, preventing less experienced users being able to run it.

```suggestion
      # elixir: "~> 1.10",
```

Reply:
Admittedly, I think I forgot to do this on a few of the recent merges.

Reply:
Makes total sense. In the `config.json` file, can it still say 1.10 is required or should we remove that too? https://github.com/exercism/v3/blob/master/docs/concept-exercises.md says language version requirements are optional.

Reply:
I think in an early-merged exercise contains the decision to have it in, but commented to be a subtle prompt to students that we are trying to build towards the latest minor version of elixir.